src/eth/TransactionObject.js

```diff
@@ -131,13 +131,22 @@ export default class TransactionObject extends TransactionLifeCycle {
     let tx;
     const startTime = new Date();
     log(`waiting for transaction ${this.hash.substring(8)}... to mine`);
-    for (let i = 0; i < 240; i++) {
-      // 20 minutes max
+    for (let i = 0; i < 24; i++) {
+      // 2 minutes max
       tx = await this._web3Service.getTransaction(this.hash);
       if ((tx || {}).blockHash) break;
       log('not mined yet');
       await promiseWait(5000);
     }
+
+    if (tx && !tx.blockHash) {
+      this._gasEstimator.transactionSpeed = 'fastest';
```

Comment:
i'm on the fence about this -- on the one hand, this is probably the right thing to do in most cases, and on the other hand, this will change the behavior for all other transactions sent from the same maker instance without telling the user. (unless they have disabled gas price in which case it does nothing)

probably better just to drop it, but to make it clear in the documentation how the user would change gas estimator settings for a live instance.

Reply:
@levity what do you think about disabling the gasPrice instead? Granted, that's still a change for all other transactions. But I think with MetaMask, this would cause all other transactions to prompt the user to select a gas price, which is probably the ideal behavior here. But if they're not using MetaMask, it could still be weird...so I could see arguments for either side.

Reply:
I'm still in favor of doing nothing and documenting options for the user. Leaning on the principle of least surprise here.